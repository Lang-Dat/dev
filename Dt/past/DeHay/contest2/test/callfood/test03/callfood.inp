7
8 1
1 1
6 2
20 15
12 6
10 7
10 16
