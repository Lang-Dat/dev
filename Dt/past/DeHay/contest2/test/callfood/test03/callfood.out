1
2
4
10
17
27
42
